### ISSUE TITLE
Apple diagnostic boot option does not work with OCLP installed

### ISSUE THREAD

**Issue by skynan22** [dortania/OpenCore-Legacy-Patcher#886]
### Machine Model

MacBookPro11,1

### Application Version

Latest Release

### Application Variant

CLI (Command Line Interface)

### Have you verified whether this issue has been opened before? If no, your issue will be closed

true

### Have you verified whether this issue is covered in our guide? ie. Troubleshooting and Legacy Acceleration pages

true

### Is this issue with a 3rd party application?

false

### What versions of macOS are you seeing the problem on?

macOS 12, Monterey

### Where does this issue happen?

Between booting macOS and Login Screen

### What is the Issue?

I want to run apple diagnostics by holding D-key at boot time but instead of the diagnostic option only recovery modus starts up. How can apple diagnostics menu be accessed at boot time, after installing MacOS Monterey 12.1 via OCLP?

**Comment by khronokernel** [comment]
Option + D should always still work as it does not have any reliance on local installations:

* https://support.apple.com/en-ca/HT202731

**Comment by skynan22** [comment]
I have already tested Option+D but it does not work for me because:
1. option+D is for M1 MacBooks only, with my Intel based mbp 11,1 it is D only, as described in the link above from khronokernel
2. since I have installed olcp the apple boot options are not available anymore, especially the diagnostics option. 

The reason why I wanted to start apple diagnostics is whether there is a hw defect or anything else that slows down my mbp at boot time (>120sec.) since I have installed oclp and MacOS 12.1 on it.
